src/Storage/ContentRequest/Listing.php

```diff
@@ -20,19 +22,23 @@ class Listing
     protected $config;
     /** @var Query */
     private $query;
+    /** @var PagerManager */
+    protected $pager;
 
     /**
      * Constructor.
      *
      * @param EntityManager $em
-     * @param Query         $query
-     * @param Config        $config
+     * @param Query $query
+     * @param Config $config
+     * @param PagerManager $pager
      */
-    public function __construct(EntityManager $em, Query $query, Config $config)
+    public function __construct(EntityManager $em, Query $query, Config $config, PagerManager $pager)
```

Comment:
For v3 we'd need to make this `PagerManager $pager = null` to cover BC (and note as such). I had intended on re-doing all this for 3.0.0 but :zipper_mouth_face: